stub-connector/src/test/java/uk/gov/ida/notification/apprule/StubConnectorAssetsAppRuleTests.java

```diff
@@ -33,6 +33,13 @@ public void shouldServeMetadata() throws IOException, URISyntaxException {
         assertThat(metadata).isEqualTo(expectedMetadata);
     }
 
+    @Test
+    public void shouldPublishMetadataSigningCertificates() throws URISyntaxException {
+        final Response response = stubConnectorAppRule.target(METADATA_CERTS_PUBLISH_PATH).request().get();
+
+        assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
```

Comment:
Should this test also confirm that the content of the response is sensible?

Reply:
We're testing it in the Gateway but sure - I'll add more assertions to this in another PR